TRIAGE_GUIDE.md

```diff
@@ -50,9 +50,9 @@ If there are questions about what to do with an issue please raise questions wit
       1. If it's a large change then issue should become an RFC or be brought to SIG-Network meeting for discussion. Ask requestor to bring issue back as RFC or start a discussion topic. Add the issue to the next SIG-Network meeting agenda, if that would be more appropriate. 
       2. If issue is for a feature request add `kind/feature`  or `kind/enhancement` as appropriate. Add `feature/networking` or `feature/cloud-service` as appropriate.
    3. Assign a reviewer, if required, to handle follow-up comments, to reproduce the issue or ask for further clarifying information.
-4. **If issue is rejected**: assign commenter to reject issue and provide reason for rejection, add `triage/declined` and close issue
-5. **If issue needs more information**: assign comment to add request for more information and add `triage/needs-information` label.
-6. **If issue is accepted**: remove `needs-triage` label, set priority for issue and add `triage/accepted` label. 
+4. **If issue is rejected**: assign reviewer to reject issue and provide reason for rejection. Add `triage/declined` and close issue
+5. **If issue needs more information**: assign reviewer to add request for more information. Add `triage/needs-information` label.
```

Comment:
> assign reviewer to add request for more information

I am confused who the "reviewer" is here. Does this mean a sig-network reviewer should be assigned to articulate the request for more info? Or do we mean to assign the issue back to the person who submitted the issue?

Reply:
reviewer is SIG-Network reviewer.  See lines above " Assign a reviewer, if required, to handle follow-up comments, to reproduce the issue or ask for further clarifying information." Tend to use _reporter_ for the person creating the issue and _reviewer_ for the person assigned to issue to ensure it can be accepted.

I could add "Assign a reviewer on behalf of SIG-Network" or "Assign an issue reviewer" above.  Or could use a different term. Such as `community member to act on SIG-Network's behalf" or similar.

Happy to take suggestions.

Reply:
The use of those terms ("reviewer" & "reporter") make sense, I was mostly wondering why we're assigning reviewers to issues that were rejected or need more info (so I thought maybe you meant to say "reporter" on those lines instead) - but your update clarifies this now.